runtime/src/accounts_db.rs

```diff
@@ -1833,7 +1840,7 @@ impl AccountsDB {
                 );
                 return 0;
             }
-            self.do_shrink_slot_stores(slot, stores.iter());
+            self.do_shrink_slot_stores(slot, stores.iter(), false);
```

Comment:
This doesn't need the argument. It can be set to `false` in the call to `store_accounts_custom` inside `do_shrink_slot_stores`